.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,170 @@
+      - [`@kiwicom/orbit-components`](#kiwicomorbit-components-)
+      - [`@kiwicom/babel-plugin-orbit-components`](#kiwicombabel-plugin-orbit-components-)
+      - [`@kiwicom/orbit-design-tokens`](#kiwicomorbit-design-tokens-)
+      - [`@kiwicom/orbit-tailwind-preset`](#kiwicomorbit-tailwind-preset-)
+      - [Root](#root)
+    - [How to develop](#how-to-develop)
+    - [Testing](#testing)
+    - [Unit tests](#unit-tests)
+    - [Visual regression test](#visual-regression-test)
+  - [Contributing to the docs](#contributing-to-the-docs)
+  - [Adding a new icon](#adding-a-new-icon)
+  - [Props naming convention](#props-naming-convention)
+  - [Component design](#component-design)
+  - [Commits](#commits)
+  - [Versioning](#versioning)
+
+---
+
+## Development Information
+
+### Requirements
+
+- [Node 14.13+](https://nodejs.org/en/)
+- [Yarn](https://yarnpkg.com/en/)
+
+### Technologies we use
+
+- [React](https://reactjs.org/docs/getting-started.html) for development
+- [Styled-components](https://www.styled-components.com/docs) for styling
+- [Jest](https://jestjs.io/) and [Testing Library](https://testing-library.com/) for testing
+- [Cypress](https://www.cypress.io/)
+- [Storybook](https://storybook.js.org/docs/react/get-started/introduction) for development
+- [Typescript](https://www.typescriptlang.org/docs) for types
+- [Gatsby](https://www.gatsbyjs.com/docs/) for docs
+
+### Repository Structure
+
+This is a monorepo containing a few Orbit packages:
+
+#### `@kiwicom/orbit-components`
+
+- `/packages/orbit-components/src` – All Source files
+- `/packages/orbit-components/es` – **GENERATED** ES compliant modules
+- `/packages/orbit-components/lib` – **GENERATED** CJS compatible modules
+- `/packages/orbit-components/config` – Build and validation scripts, test configuration
+- `/packages/orbit-components/.storybook` – Storybook configuration
+
+#### `@kiwicom/babel-plugin-orbit-components`
+
+- `/packages/babel-plugin-orbit-components/index.js` – Source file
+- `/packages/babel-plugin-orbit-components/scripts` – Test scripts
+
+#### `@kiwicom/orbit-design-tokens`
+
+- `/packages/orbit-design-tokens/src` – Source files
+- `/packages/orbit-design-tokens/dist` – **GENERATED** Design tokens
+
+#### `@kiwicom/orbit-tailwind-preset`
+
+- `/packages/orbit-tailwind-preset/src` – Source files
+- `/packages/orbit-tailwind-preset/dist` – **GENERATED** Tailwind preset
+
+#### Root
+
+- `/flow-typed` – Flow library definitions
+- `/scripts` – Scripts for development and publishing
+
+### How to develop
+
+Install dependencies:
+
+```sh
+yarn install
+```
+
+Build the design tokens and the tailwind preset:
+
+```sh
+yarn tokens build
+yarn tailwind-preset build
+```
+
+For developing Orbit, components run Storybook:
+
+```sh
+yarn dev
+```
+
+If you would like to try out the development version of one of our packages, we recommend using [yalc](https://github.com/wclr/yalc):
+
+```sh
+yarn global add yalc
+```
```

Comment:
This refers on how to install a global dependency. Should we be responsible for documenting it? Or at least say something like
> If you don't have it installed, consider installing it globally: `yarn global add yalc`

💭  just a suggestion, non-blocking.